editor/src/com/talosvfx/talos/editor/addons/scene/SceneEditorWorkspace.java

```diff
@@ -1187,15 +1187,15 @@ protected boolean canMoveAround () {
 			return true;
 		}
 
+		if(entityUnderMouse != null && entityUnderMouse.isEditorTransformLocked()) {
+			return true;
+		}
+
 		//if(gizmo != null && !(gizmo instanceof GroupSelectionGizmo) && gizmo.getGameObject().isEditorTransformLocked()) {
 		if(gizmo != null) {
```

Comment:
at this point gizmo is returning null, because GO is not active

Reply:
@vuvkar does this also fix the case where you can click through? For example, we have a big item, its transform is locked. But we click something also in the bounds of that, sometimes in the foreground so it should be clicked. But for some reason for the background item that is locked, it prevents us selecting anything else in front of it?